tests/runtests.py

```diff
@@ -44,31 +53,38 @@ def upath(path):
     'django.contrib.messages',
     'django.contrib.admin',
     'django.contrib.staticfiles',
-    "mezzanine.boot",
-    "mezzanine.conf",
-    "mezzanine.core",
-    "mezzanine.generic",
-    "mezzanine.pages",
-    "mezzanine.forms",
-    'django.contrib.comments',
-    'filebrowser_safe',
-    'grappelli_safe',
+]
+
+if is_mezzanine_installed():
```

Comment:
different settings file (and maybe `test/contrib`) sounds like a very good idea.